contents/scalability/scalable-development.md

```diff
@@ -1,6 +1,6 @@
 # Scalable Development - An Introduction
 
-Authors: Loh Jia Shun Kenneth
+Authors: Loh Jia Shun Kenneth, Vivek Lakshmanan
 
 ## Scalable Development
 When Pokémon Go first launched in July 6, the heavy traffic from users caused its servers to crash. However, as of 2017, Google handles at least 3.5 billion searches daily, Whatsapp handles at least 10 billion messages a day, and Facebook hosts 1.2 billion daily active users. What makes these companies different?
```

Comment:
I know this part was not touched in the commit, but the example of Pokémon Go didn't link very well to the point. Is it because the heavy traffic far exceeded that of Facebook/Google, or because their infrastructure was not scalable? Furthermore no numbers were given so it is are to link it to the point